samples/eshop/infra/infra.bicep

```diff
@@ -115,3 +115,9 @@ output servicebus string = servicebus.name
 @description('Event Bus connection string')
 #disable-next-line outputs-should-not-contain-secrets
 output eventBusConnectionString string = (AZURESERVICEBUSENABLED == 'True') ? servicebus.listSecrets().connectionString : rabbitmq.properties.host
+
+@description('Envoy image name')
+output envoyImageName string = 'eshop-envoy'
+
+@description('Envoy image tag')
+output envoyImageTag string = 'v1.32.3'
```

Comment:
Shouldn't `eshop-envoy` have a different tag than the `envoyproxy/envoy`?

Reply:
we can choose whichever tag we want, I left it as this one for consistency. which tag name do you think we should use?